README.md

```diff
@@ -22,13 +22,44 @@ Once created don't forget to update the:
 - Jetpack Compose setup ready to use. 
 - Sample Espresso, Instrumentation & JUnit tests.
 - 100% Gradle Kotlin DSL setup.
+- Pre-commit script to ensure gradle `check` task passes before every commit
 - CI Setup with GitHub Actions.
 - Publish to **Maven Central** with Github Actions.
 - Dependency versions managed via `buildSrc`.
 - Kotlin Static Analysis via `detekt` and `ktlint`.
 - Issues Template (bug report + feature request).
 - Pull Request Template.
 
+## Known issues
+
+# agp incompatibility with IntelliJ IDEA
+
+Opening this template in IDEA might result in an error like this:
+
+```
+The project is using an incompatible version (AGP 8.0.0) of the Android Gradle plugin. Latest supported version is AGP 7.4.0
+```
+
+This is caused by IDEA support for AGP lagging behind releases of AGP and is tracked by bugs like https://youtrack.jetbrains.com/issue/IDEA-317997.
+
+To get this working in the mean time, you can downgrade the version of `agp` in `libs.versions.toml` to a supported version. 
+
+# Use correct JVM version
```

Comment:
I'm unsure about this paragraph.

As you're using a jvmToolchain, Gradle will download and install the right version of the JDK which is needed for this project (17)

I believe there was a bug in AGP which resulted in the toolchain from being ignored, and should be fixed in AGP 8.1+

Reply:
I'm using Gradle 8.1.1 (checked by running `./gradlew --version` at the command line) and I get the error `Invalid value (20) passed to --jvm-target` if I have java 20 in my path and I don't get this error if I have java 17 in my path.

Reply:
This is an issue with Detekt and not this template itself, so please remove

Reply:
This is an issue which I hit when trying to use this template and so I think it's helpful to have a workaround documented in the README. Of course you're the owner of this template so I'll remove this if you disagree but I want to double check - are you sure you want this to be removed?